api/v1/helpers/verbs/doPut.js

```diff
@@ -70,9 +70,10 @@ function doPut(req, res, next, props) {
             // take nullified fields out of changed fields
             o.changed(key, false);
           } else {
-
-            // value may have changed. set changed to true to
-            // trigger checks in the model
+            /**
+             * value may have changed. set changed to true to
+             * trigger checks in the model
+             */
             o.changed(key, true);
```

Comment:
I did not change line 77, just corrected the formatting for the comments.  Need to check why line 77 was added.